MovieApp/MovieListViewController.swift

```diff
@@ -9,31 +9,65 @@ import Foundation
 import SnapKit
 import UIKit
 import MovieAppData
+import Network
 
 
 protocol SearchBoxActions: AnyObject {
-    func showResults(query: String)
-    func showMainPage()
+    func userTyped(textBoxText: String)
+    func userClearedText()
+    func userPressedCancel()
 }
 
+
+protocol MovieCollectionViewActions: AnyObject {
```

Comment:
It would be more readable for me to name this MovieCollectionViewDelegate so that it's clear from the name that it is the delegate.